app/src/main/java/com/example/workoutapp/MainActivitytwo.kt

```diff
@@ -114,8 +117,10 @@ class MainActivity : ComponentActivity() {
     private fun <E> MutableList<E>.add(element: MutableLiveData<List<E>>) {
 
     }
-
-
+    private fun getCurrentDateAndTime(): String {
+        val sdf = SimpleDateFormat("'Today is:\n'dd-MM-yyyy")
+        return sdf.format(Date())
+    }
```

Comment:
Will this be used/invoked from anywhere else?

If not... two lines may not be worth refactoring to a separate method.